components/utilities/object.js

```diff
@@ -0,0 +1,21 @@
+(function(d3, fc) {
```

Comment:
Another good spot, I didn't even realise we had committed to a particular
set of browsers. Can you raise an issue to revisit the supported browser
list and for now it's fine to assume Object.keys support?

On Mon, 20 Apr 2015 08:45 Jan Akerman <notifications@github.com> wrote:

> In components/utilities/object.js
> <https://github.com/ScottLogic/d3-financial-components/pull/299#discussion_r28668914>
> :
>
> > @@ -0,0 +1,21 @@
> > +(function(d3, fc) {
>
> Yeah, the README says that the components are Firefox 3+ and Safari 3.2,
> but I don't think Object.Keys was supported until version 4 and 5
> respectively.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ScottLogic/d3-financial-components/pull/299/files#r28668914>
> .
>